### ISSUE TITLE
InputfieldIcon::render() - previously removed prefix will be added again

### ISSUE THREAD

**Issue by kixe** [processwire/processwire-issues#1002]
### Short description of the issue

If the Inputfield property: `'prefixValue'` is set to **false** the prefix will be removed by function `InputfieldIcon::getAttributes()` but added again in function `InputfieldIcon::render()`

### Expected behaviour
The prefix should not be added again and the value should be saved without a prefix.

Removed here:
https://github.com/processwire/processwire/blob/22fe5bf9ef2dbe05fd8d6889810ae367a333fbef/wire/modules/Inputfield/InputfieldIcon/InputfieldIcon.module#L84-L85

Added/ Readded here:
https://github.com/processwire/processwire/blob/22fe5bf9ef2dbe05fd8d6889810ae367a333fbef/wire/modules/Inputfield/InputfieldIcon/InputfieldIcon.module#L38

* ProcessWire version: 1.0.142
* (Optional) PHP version: 7.2
* (Optional) MySQL version: 
* (Optional) Any 3rd party modules that are installed and could be related to the issue:

**Comment by ryancramerdesign** [comment]
@kixe I can see that it is removing and adding the prefix in the places you linked, but it looks to me like that's intended? It's been awhile since I've worked with this field, so I might not understand what you mean. The prefixValue property seems to indicate it applies to the external value only, so it looks to me like it's supposed to keep using it internally, unless it's resulting in some error. Are you observing an error, and if so, can you describe it further? (and how to reproduce if applicable)

**Comment by kixe** [comment]
After closer examination, I have found out that the option to remove the prefix is ​​relevant only if the Inputfield is used in conjunction with module configurations, like in the module **SystemNotifications**. The prefixValue property works here. (internal?)

However, in my use case, the value is stored in a page field of type **FieldtypeText**. The input field for this is added via hook in `ProcessPageEdit::buildFormSettings()` under the tab: **Settings** in the edit area of ​​the respective page.

Since the selectable option value ​​always include the prefix, the desired change would have anyway no effect.

I fixed the problem by removing the prefix with a hook in `Page::saveReady()`.

To take a closer look at what I mean, find the related module here: [PageIcon](https://github.com/kixe/PageIcon)

The ticket can be closed.
